src/static/webpack/rules/jsLoader.js

```diff
@@ -32,7 +32,9 @@ export default function({ config, stage }) {
 
   return {
     test: /\.(js|jsx|mjs)$/,
+    include: [config.paths.SRC, `${config.paths.DIST}/react-static-routes.js`],
     use: [
+      // 'thread-loader',
```

Comment:
Does `thread-loader` not work properly here?

Reply:
it threw some error that i didnt found:

```
Module build failed (from node_modules/thread-loader/dist/cjs.js):
Thread Loader (Worker 1)
.presets[0][0] must be a string, object, function
    at Array.forEach (<anonymous>)
    at Array.forEach (<anonymous>)
```

but it is:

```js
      {
        loader: 'babel-loader',
        options: {
          babelrc: false,
          configFile: false,
          compact: false,
          presets: [
            [
              babelPreset,
              { helpers: true },
            ],
          ],
          cacheDirectory: true,
          sourceMaps: false,
        },
      },
```

so im not sure where the problem is. the docs stated anyway that its only useful for large projects (starting a worker gives 200ms overhead), so if we want to use it, it should be some optin strategy i think